tests/helperfunctions.q

```diff
@@ -0,0 +1,25 @@
+startorstopproc:{[startorstop;procname;processcsv] 
+	.proc.sys getenv[`TORQHOME],"/torq.sh ",startorstop," ",procname," -csv ",processcsv
+	};
+
+deadproccheck:{[proctype;procname]
+	/ pairs are of the form "<PID> ssh" for 'PID TTY...' part of pgrep, and "<PID> q" if process is running
+	pidnamepairs:.proc.sys "pgrep -lf \"stackid ",getenv[`KDBBASEPORT]," -proctype ",proctype," -procname ",procname,"\"";
+	not "q" in last each pidnamepairs
+	};
+
+opentorqhandle:{[port]
+	hopen `$"::",port,":admin:admin"	
```

Comment:
Could you change the password to a variable e.g. testuserpass:"admin:admin" or something. 
Then, when people change the default passwords they just have to change the variable in one of the csv files or other files that they load rather than having to overwrite this functino

Reply:
assuming test processes will have a `proctype` that's already got a defined username/pass, could just make this a wrapper of `.servers.opencon` (which accepts an hsym of `` `:host:port``) & automatically use the "right" username/pass for that proctype?

Reply:
Yeah... good point. I think both maybe have value? Perhaps two methods, along the lines of openbyname and openbyport. If we use port then we are potentially hard coding port numbers into the tests which isn't good. But if we do openbyname then we are requiring that every process that the test process may need to connect to needs to have it in the accesslist, which is also a bit of a pain. But two ways of doing it is probably ok.

Reply:
So on closer inspection this function isn't actually being used anywhere. Instead connections to procs are done by calling `.servers.startup` and then using the `gethandle` function. Will I just get rid of this or shall I incorporate these suggestions in `gethandle`?

Reply:
I think gethandle is fine and remove this one then